EIPS/eip-2645.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 2645
+title: Hierarchical Deterministic Wallet for Layer-2
+author: Tom Brand (tom@starkware.co), Louis Guthmann (louis@starkware.co)
+discussions-to: https://ethereum-magicians.org/t/hierarchical-deterministic-wallet-for-computation-integrity-proof-cip-layer-2/4286
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-05-13
+---
+
+## Simple Summary
+In the context of Computation Integrity Proof (CIP) Layer-2 solutions such as ZK-Rollups, users are required to sign messages on new elliptic curves optimized for those environnements. We leverage existing work on Key Derivation ([BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki), [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) and [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)) to define an efficient way to securely produce CIP L2s private keys, as well as creating domain separation between Layer-2 applications.
```

Comment:
This is much too long for Simple Summary.  Think of what you would put in an email subject line, or a forum thread topic, or a GitHub PR title.